src/transformStateWithClones.js

```diff
@@ -7,7 +7,32 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  let status = Object.assign({}, state);
```

Comment:
Assign it to a constant. Also I would like to see some better naming
```suggestion
  const stateCopy = Object.assign({}, state);
```